### ISSUE TITLE
Keyboard with Bundled Assertions

### ISSUE THREAD

**Issue by ravenokko** [shopware/platform#10892]
### Technical TODO

Adapt the RegisteredUserBuysProduct.spec.ts to only navigate and interact with the storefront using a keyboard, including the [custom assertion: toHaveVisibleFocus](https://github.com/shopware/shopware/issues/10891), and bundled with a collection of assertions to cover missing [actionability checks](https://playwright.dev/docs/actionability#introduction)
